No real point to this code and it's been dropped in 3.4.x so let's punt it 
in 3.3.x too.  Fix cross-compiling which is good.

http://bugs.gentoo.org/show_bug.cgi?id=79487

--- gcc-3.3.5/configure.in
+++ gcc-3.3.5/configure.in
@@ -902,36 +902,6 @@
   fi
 fi
 
-# Handle ${copy_dirs}
-set fnord ${copy_dirs}
-shift
-while test $# != 0 ; do
-  if test -f $2/COPIED && test x"`cat $2/COPIED`" = x"$1" ; then
-    :
-  else
-    echo Copying $1 to $2
-
-    # Use the install script to create the directory and all required
-    # parent directories.
-    if test -d $2 ; then
-      :
-    else
-      echo >config.temp
-      ${srcdir}/install-sh -c -m 644 config.temp $2/COPIED
-    fi
-
-    # Copy the directory, assuming we have tar.
-    # FIXME: Should we use B in the second tar?  Not all systems support it.
-    (cd $1; tar -cf - .) | (cd $2; tar -xpf -)
-
-    # It is the responsibility of the user to correctly adjust all
-    # symlinks.  If somebody can figure out how to handle them correctly
-    # here, feel free to add the code.
-
-    echo $1 > $2/COPIED
-  fi
-  shift; shift
-done
 
 # Work in distributions that contain no compiler tools, like Autoconf.
 tentative_cc=""
